src/components/SQFormScrollableCard/SQFormScrollableCard.tsx

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
-import {
-  Card,
-  CardHeader,
-  CardContent,
-  CardActions,
-  Grid,
-  makeStyles,
-} from '@material-ui/core';
+import {Card, CardHeader, CardContent, CardActions, Grid} from '@mui/material';
 import {Formik, Form} from 'formik';
 import {useDebouncedCallback} from 'use-debounce';
 import SQFormButton from '../SQForm/SQFormButton';
 import SQFormHelperText from '../SQForm/SQFormHelperText';
 import {useInitialRequiredErrors} from '../../hooks/useInitialRequiredErrors';
 import {HEADER_HEIGHT} from '../../utils/constants';
-import type {CreateCSSProperties} from '@material-ui/core/styles/withStyles';
-import type {TypographyVariant, GridProps} from '@material-ui/core';
+import type {GridProps} from '@mui/material';
 import type {FormikHelpers, FormikValues} from 'formik';
 import type {AnyObjectSchema} from 'yup';
 
 export type SQFormScrollableCardProps<Values extends FormikValues> = {
-  /** An object of css-in-js style properties to be passed and spread onto `classes.cardContent` */
-  cardContentStyles?: CreateCSSProperties;
+  /** An object of css-in-js style properties to be passed and spread onto the `CardContent` component */
+  cardContentStyles?: Record<string, unknown>;
```

Comment:
I think we should use the `SxProps` type for these. 
`import { SxProps } from '@mui/material'`
or if that doesn't work, then try 
`import { SxProps } from '@material-ui/system';`

found here: https://github.com/mui/material-ui/issues/27564

Reply:
Yeah that's a better idea.  I'll change that.